src/graph/finding-negative-cycle-in-graph.md

```diff
@@ -78,3 +78,5 @@ Tasks in the online judges
 
 The list of tasks that need to search for a cycle of negative weight:
 * [UVA: Wormholes](https://uva.onlinejudge.org/index.php?option=com_onlinejudge&Itemid=8&page=show_problem&problem=499)
```

Comment:
Please add a newline before the first list item. Otherwise the list doesn't get rendered as a list